bower.json

```diff
@@ -20,6 +20,6 @@
   },
   "devDependencies": {
     "purescript-psci-support": "^4.0.0",
-    "purescript-quickcheck": "^5.0.0"
+    "purescript-quickcheck": "^6.1.0"
```

Comment:
The previous version was causing a compile-time error in `purescript-typelevel-prelude`.